modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/DiagramBuilder.js

```diff
@@ -253,9 +254,13 @@ export default function DiagramBuilder({version}) {
 
 	const contextProps = {
 		collidingElements,
+		elementRectangle,
+		elements,
```

Comment:
@cgmonte you just need to remove elements and setElements from the context props. The rest is all good :)

Reply:
@naraandrade done :slightly_smiling_face: